extensions/wikia/ArticleVideo/scripts/article-featured-video.js

```diff
@@ -84,22 +89,42 @@ require([
 				};
 
 			if (ooyalaAdSetProvider.canShowAds()) {
-				options.adSet = ooyalaAdSetProvider.get(1, correlator, {
-					contentSourceId: videoData.dfpContentSourceId,
-					videoId: videoId
-				});
 				options.replayAds = ooyalaAdSetProvider.adsCanBePlayedOnNextVideoViews();
+
+				if (a9 && context.bidders && context.bidders.a9Video) {
```

Comment:
it's the same check as in `isVideoBidderEnabled` - could we reuse that function?

Reply:
No, we can't, it's behind factory interface :(